app/src/test/java/com/kickstarter/viewmodels/PrivacyViewModelTest.kt

```diff
@@ -0,0 +1,103 @@
+package com.kickstarter.viewmodels
+
+import com.kickstarter.KSRobolectricTestCase
+import com.kickstarter.factories.UserFactory
+import com.kickstarter.libs.MockCurrentUser
+import com.kickstarter.models.User
+import org.junit.Test
+import rx.observers.TestSubscriber
+
+class PrivacyViewModelTest : KSRobolectricTestCase() {
+    private lateinit var vm: PrivacyViewModel.ViewModel
+
+    private val currentUserTest = TestSubscriber<User>()
+    private val hideConfirmFollowingOptOutPrompt = TestSubscriber<Void>()
+    private val showConfirmFollowingOptOutPrompt = TestSubscriber<Void>()
+
+    private fun setUpEnvironment(user: User) {
+        val currentUser = MockCurrentUser(user)
+        val environment = environment().toBuilder()
+                .currentUser(currentUser)
+                .build()
+
+        this.vm = PrivacyViewModel.ViewModel(environment)
+
+        currentUser.observable().subscribe(this.currentUserTest)
+        this.vm.outputs.hideConfirmFollowingOptOutPrompt().subscribe(this.hideConfirmFollowingOptOutPrompt)
+        this.vm.outputs.showConfirmFollowingOptOutPrompt().subscribe(this.showConfirmFollowingOptOutPrompt)
+    }
+
+    @Test
+    fun tesCurrentUser() {
+        val user = UserFactory.user()
+
+        setUpEnvironment(user)
+
+        this.currentUserTest.assertValue(user)
+    }
+
+    @Test
+    fun testSettingsViewModel_optIntoFollowing() {
```

Comment:
Are we going to use camelCase or snake_case for our testing methods? I've been using camelCase.

Reply:
I renamed this one because it's not even in the SettingsViewModel anymore but I try to write test methods in the format `testAnOutput_whenAnInput`, so I have a test `testHideConfirmFollowingOptOutPrompt_userConfirmsOptOut` that's testing if the `hideConfirmFollowingOptOutPrompt` emits nothing when the user confirms that they want to opt out.